x-pack/solutions/security/plugins/security_solution/public/onboarding/components/onboarding_body/cards/common/card_selector_list.test.tsx

```diff
@@ -40,53 +51,89 @@ describe('CardSelectorList', () => {
     Element.prototype.scrollIntoView = scrollIntoViewMock;
   });
   beforeEach(() => {
+    mockExperimentalFeatures.mockReturnValue({});
     jest.clearAllMocks();
   });
   afterAll(() => {
     jest.useRealTimers();
   });
 
   it('renders the component with the correct title', () => {
-    const { getByText } = render(<CardSelectorList {...defaultProps} />);
+    const { getByText } = render(
+      <TestProviders>
+        <OnboardingContextProvider spaceId="default">
+          <CardSelectorList {...defaultProps} />
+        </OnboardingContextProvider>
+      </TestProviders>
+    );
```

Comment:
NIT: The `render` function accepts a `wrapper` setting. Sometimes, it's more convenient to create the `wrapper` outside the test and reuse it in the test cases:

```ts
const wrapper: React.FC<PropsWithChildren<{}>> = ({ children }) => (
  <TestProviders>
    <OnboardingContextProvider spaceId="default">{children}</OnboardingContextProvider>
  </TestProviders>
);
```

And then:

```suggestion
    const { getByText } = render(<CardSelectorList {...defaultProps} />, { wrapper });
```